test_data/import_publications.py

```diff
@@ -59,71 +62,76 @@ def map_user_to_arp():
                     arp_dict[user['username']]['scn'] = query_response.json(
                     )[0]['id']
 
+
 def upload_file(bearer_token):
     print('upload file...')
     headers = {
-      'Authorization': 'Bearer {}'.format(bearer_token),
-      'accept': 'application/pdf'
+        'Authorization': 'Bearer {}'.format(bearer_token),
+        'accept': 'application/pdf'
     }
     # create
     print('create...')
     response = requests.post(
-      upload_create,
-      json={
-        'filename': 'test_file.pdf',
-        'size': 32404,
-        'lastmodified': 1353189358000,
-        'mimetype': 'application/pdf'
-      },
-      headers=headers)
+        upload_create,
+        json={
+            'filename': 'test_file.pdf',
+            'size': 32404,
+            'lastmodified': 1353189358000,
+            'mimetype': 'application/pdf'
+        },
+        headers=headers)
     uploadId = response.json()['uploadId']
     key = response.json()['key']
     # prepare
     response = requests.post(
-      upload_prepare,
-      json={
-        'number': '1',
-        'uploadId': uploadId,
-        'body': str(test_file),
-        'key': key
-      },
-      headers=headers)
+        upload_prepare,
+        json={
+            'number': '1',
+            'uploadId': uploadId,
+            'body': str(test_file),
+            'key': key
+        },
+        headers=headers)
     presignedUrl = response.json()['url']
     # upload
-    response = requests.put(presignedUrl, headers = { 'Accept': 'appliation/pdf' }, data = test_file)
+    response = requests.put(presignedUrl, headers={
+                            'Accept': 'appliation/pdf'}, data=test_file)

```

Comment:
litt rart linjeskift her?
```suggestion
    response = requests.put(presignedUrl, headers={ 'Accept': 'appliation/pdf'}, data=test_file)

```